main.tf

```diff
@@ -0,0 +1,200 @@
+// Define Local Vars
+
+locals {
+  tags = {
+    Terraform   = "${var.terraform_tag}"
+    Environment = "${var.environment}"
+  }
+}
+
+data "aws_region" "current" {}
+
+// VPC Configuration
+
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~>3.18.1"
+
+  name = "${var.prefix}-spot-instance-vpc-${var.environment}"
+  cidr = var.use_ipam_pool ? null : var.vpc_cidr_block
+
+  azs            = ["${data.aws_region.current.name}a", ]
+  public_subnets = [var.public_subnet_cidr_block, ]
```

Comment:
done

Reply:
Does terraform fmt pick up "${var.XXX}" and throw them as errors?

Because I keep seeing this in your code but you are only meant to use var.XXX now no strings